src/components/rolo-card.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, customElement, html, property, css } from 'lit-element';
+
+@customElement('rolo-flip-card')
+class RoloCard extends LitElement {
+  @property({
+    type: Boolean,
+    attribute: 'is-flipped',
+    converter: (val) => /^(true)$/.test(val),
```

Comment:
Interesting, so this is a property converter. I like it. I'm going to steal it. Not sure how or when I'll use it but I'm sure I'll figure that out down the line.

Reply:
I know, right?! Actually, converter can be a function, as above, or it can be an object with methods `toAttribute` or `fromAttribute`. So, you actually have even more granular control of how your component interacts with its data. 
https://lit-element.polymer-project.org/guide/properties#conversion-converter 
@Plutonaught

Reply:
Oh, I meant to mention, when it's a function, as above, it defaults to `fromAttribute`; it's a function that takes two parameters: `value` and `type`(optional), where value is the attribute's value and the type is the result of typeof value, (i think). I didn't see much use in the type parameter.